docs/dev-guide/integration/rest-api/tasks.rst

```diff
@@ -184,6 +184,28 @@ Task Group Management
 
 .. _task_group_summary:
 
+Cancelling Tasks in Task Group
+------------------------------
+
+All asynchronous tasks in a particular task group may be canceled by the user before they complete. A
+task must be in the *waiting* or *running* states in order to be canceled.
+
+.. Note::
+
+   It is possible for a task to complete or experience an error before the cancellation request is
+   processed, so it is not guaranteed that a task's final state will be 'canceled' as a result of
+   this call. In these instances this method call will still return a response code of 200.
```

Comment:
@dkliban  I have tried to keep it in consistent with the tasks API. Tasks return 200, I will see more about it.